### ISSUE TITLE
Unable to proceed to publish a  new project after selecting add tags

### ISSUE THREAD

**Issue by Dumzy006** [unstructuredstudio/zubhub#879]
This issue is similar to the issue I raised in #874

@srish I can't proceed to publish a new project after completing all steps when I click on add tags
Then select hashtag that describes my project and I am not getting any error message as to why I can't proceed
See screenshot below at @srish 
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/f4162cd8-df10-44d5-8dce-85f94b543e4f)

**Comment by Dumzy006** [comment]
I also tried to publish without a tag still the same issue
I can't move pass this screen
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/6eff42dd-df79-400f-a88d-20ea3b0a0c4a)

**Comment by Temabo** [comment]
@Dumzy006 this is a duplicate of the issue you created in #874

**Comment by Dumzy006** [comment]
Hello @Temabo 
That issue is for my drafts and this is for create project

**Comment by coderatomy** [comment]
Am wrapping these into one PR @Temabo @Dumzy006

**Comment by coderatomy** [comment]
Alright. So what's happening here is that we actually need either of the below
- A video to upload
- A video link 
- Or an image
...for a project to be made

![image](https://github.com/unstructuredstudio/zubhub/assets/141822315/2cf31d5d-ce70-435a-81e9-a8507c6014e0)

The only thing to add here are validation checks such that a user can't continue to step 3 without providing either of the 3

**Comment by Dumzy006** [comment]
Hello @coderatomy 
I feel the user should see an error message once he tries to move to the next step stating why he can't proceed until he fulfils either of the 3
See screenshots of what I came up with

User sees an error message once he tries to proceed when none of the 3 has been provided
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/1c155c09-27d1-401f-adcc-5d127bcc8615)

once either of the 3 is provided, the error message goes away
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/f4ab32f7-c6f9-483b-9604-4b3f9b781902)

**Comment by Dumzy006** [comment]
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/abff56d5-9f6b-409b-8305-865f63f13768)

**Comment by Dumzy006** [comment]
Hello @coderatomy rather than having error messages displayed as text in the screenshot I provided earlier  we can have it in an alert form.. What do you think @srish 
see screenshot below
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/3ab618a8-fc73-4a86-b337-6c3ff8ee1578)
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/3664e55b-1700-444a-a3c1-474bdc18d54a)

**Comment by srish** [comment]
Thanks for filing this! Please see my comment on this task: https://github.com/unstructuredstudio/zubhub/issues/797. Closing for now.
